.go-version

```diff
@@ -1 +1 @@
-1.22.3
+1.22.4--alpine3.20
```

Comment:
This file is used by people using goenv, to switch versions, can you revert this and only update alpine in dockerfile?
```suggestion
1.22.3
```

Reply:
I'm sorry, I've never used Goenv before and didn't know that.
Can I change the version in the file to 1.22.4 and update Alpine in Dockerfile?